articles/architecture-scenarios/implementation/b2b/b2b-authentication.md

```diff
@@ -0,0 +1,133 @@
+---
+title: Authentication
+description: Understand how authentication works in your B2B IAM implementation.
+toc: true
+topics:
+    - b2b
+    - b2biam
+    - authentication
+    - universal-login
+contentType: concept
+useCase:
+    - authentication
+---
+# Authentication
+
+<%= include('../../_includes/_authentication/_introduction.md', { platform: 'b2b' }) %>
+
+Often companies need to segregate their users by organization and sometimes users can have access to more than one organization.  Knowing which of these scenarios is relevant to your company will help define how to determine in which connection a user exists: whether you need to do it, when you need to do it, and how to accomplish it.  See [Home Realm Discovery](#home-realm-discovery) to determine if this is something relevant to your company.
```

Comment:
We should eventually move this to the _introduction include to be consistent, but that seems too painful to do right now.

Reply:
Done